packages/landing/src/attribution.ts

```diff
@@ -132,4 +132,12 @@ export class MapAttribution {
     const bbox: BBox = [swCoord.lon, swCoord.lat, neCoord.lon, neCoord.lat];
     return bbox;
   }
+
+  /**
+   * Add attribution for vector map
+   */
+  vectorAttribution(): void {
+    this.attributionControl = new maplibre.AttributionControl({ compact: false });
```

Comment:
need to call `this.removeAttribution` to cleanup any existing attribution controls

Reply:
Good pick, I forget to testing switch button.